pkg/awsvpc/nacl_analysis.go

```diff
@@ -98,8 +99,8 @@ func (na *AWSNACLAnalyzer) GetNACLRule(index int) (ruleStr string, ruleRes *comm
 		direction = commonvpc.Inbound
 	}
 	ruleRes = &commonvpc.NACLRule{Src: src, Dst: dst, Connections: conns, Action: action}
-	ruleStr = fmt.Sprintf("index: %d, direction: %s ,cidr: %s, conn: %s, action: %s\n",
-		index, direction, ip, connStr, action)
+	ruleStr = fmt.Sprintf("ruleNumber: %d, direction: %s ,cidr: %s, conn: %s, action: %s\n",
+		ruleNumber, direction, ip, connStr, action)
```

Comment:
Move `conn` to be the last item

Reply:
done (I thought we agreed to postpone it to #767)